tests/components/netatmo/conftest.py

```diff
@@ -81,11 +81,11 @@ async def mock_entry_fixture(hass, config_entry):
 @pytest.fixture(name="sensor_entry")
 async def mock_sensor_entry_fixture(hass, config_entry):
     """Mock setup of sensor platform."""
-    with selected_platforms(["sensor"]):
+    with patch("time.time", return_value=TEST_TIME), selected_platforms(["sensor"]):
         await hass.config_entries.async_setup(config_entry.entry_id)
 
-    await hass.async_block_till_done()
-    return config_entry
+        await hass.async_block_till_done()
+        return config_entry
```

Comment:
If we want the patch to stay active for the test we should yield the config entry instead of returning it.